fuzzers/fuzzbench_forkserver/src/main.rs

```diff
@@ -349,19 +348,17 @@ fn fuzz(
 
         let cmplog_observer = StdCmpValuesObserver::new("cmplog", cmpmap, true);
 
-        let cmplog_forkserver = ForkserverExecutor::builder()
+        let cmplog_executor = ForkserverExecutor::builder()
```

Comment:
We could add an API to clone the og forkserver with a different binary (or even the same)